migration-engine/connectors/sql-migration-connector/src/sql_destructive_change_checker/destructive_change_checker_flavour/postgres.rs

```diff
@@ -101,6 +117,7 @@ impl DestructiveChangeCheckerFlavour for PostgresFlavour {
                 step_index,
             )
         } else {
+            //todo this is probably due to a not castable type change. we should give that info in the warning
```

Comment:
:+1:

I think the right way to approach these kinds of issues is to have a concrete case where we could render a better error message, put that in an issue and treat it as an improvement. It's hard to think about error ergonomics when you are in the middle of the code with all the case analysis.